docs/documentation/functions/aria-suite/aria-operations-networks/Remove-AriaNetworksLdapConfiguration.md

```diff
@@ -0,0 +1,25 @@
+# Remove-AriaNetworksLdapConfiguration
+
+## Synopsis
+
+Removes the LDAP configuration from VMware Aria Operations for Networks.
+
+## Syntax
+
+```powershell
+Remove-AriaNetworksLdapConfiguration
+```
+
+## Description
+
+The Remove-AriaNetworksLdapConfiguration cmdlet allows a user to remove the LDAP configuration from VMware Aria Operations for Networks.

```

Comment:
```suggestion
The `Remove-AriaNetworksLdapConfiguration` cmdlet allows a user to remove the LDAP configuration from VMware Aria Operations for Networks.

```

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/42261f7bbfc80fe8925418f1c844be4895a4d95f